modules/farm_fd2_school/05-Vue3/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Shopping List App</title>
+    <link
+      rel="stylesheet"
+      href="main.css"
+    />
+  </head>
+  <body>
+    <div id="shopping-list">
+      <div class="header">
+        <h1>{{header || 'Welcome'}}</h1>
+        <button
+          v-if="editing"
+          v-on:click="doEdit(false)"
+          class="btn btn-cancel"
+        >
+          Cancel
+        </button>
+        <button
+          v-else
+          v-on:click="doEdit(true)"
+          class="btn btn-primry"
+        >
+          Add Item
+        </button>
+      </div>
+      <div
+        v-if="editing"
+        class="add-item-form"
+      >
+        <input
+          v-on:keyup.enter="saveItem"
+          v-model="newItem"
+          type="text"
+          placeholder="Add an item"
+        />
+        <label>
+          <input
+            type="checkbox"
+            v-model="newItemHighPriority"
+          />
+          High Priority
+        </label>
+        <button
+          v-on:click="items.push({id: items.length + 1, label: newItem})"
```

Comment:
The `v-on` here should call the `saveItem` method.  This eliminates the duplicate code, which can be a source of maintenance errors.